app/src/org/commcare/activities/FormEntryActivity.java

```diff
@@ -2062,7 +2062,9 @@ public boolean onScroll(MotionEvent e1, MotionEvent e2, float distanceX, float d
 
     @Override
     public void advance() {
-        next();
+        if (!questionsView.isQuestionList() && canNavigateForward()) {
```

Comment:
Oh, hmm. That seems bad then because the auto advance grid widget definitely looks like it's counting on that not happening